examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    calibrations = device.native_gate_calibration
</pre><p>Why do we name this property with the singular <code>calibration</code> when it can return multiple calibrations?</p><p>Why do we name this property <code>nativate_gate_calibrations</code> instead of <code>gate_calibrations</code>? It's nicer to not have to type out long names, but if there's a possibility of the device having other gate calibrations in addition to native gate calibrations then it makes sense. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>